src/module/actor/roll-context/damage.ts

```diff
@@ -16,8 +16,7 @@ class DamageContext<
     constructor(params: DamageContextConstructorParams<TSelf, TStatistic, TItem>) {
         super(params);
         // In case the user rolled damage from their sheet, try to fish out the check context from chat
-        params.checkContext ??= this.#findMatchingCheckContext();
-
+        params.checkContext ??= this.#findMatchingCheckContext(params.options);
         if (params.outcome) params.options.add(`check:outcome:${sluggify(params.outcome)}`);
```

Comment:
I'm wondering if this was supposed to be the value of `checkContext.outcome`.

Reply:
Likely not